sdk/batch/microsoft-azure-batch/CHANGELOG.md

```diff
@@ -1,14 +1,15 @@
 # Release History
 
-## 8.1.0 (2021-01-05)
-
-- Exposes a BatchClient factory method
+## 9.0.0 (2021-01-08)
 
 ### Features
 
 - Adds support for task slots
   - `JobOperations.getTaskSlotCounts()` returns task slot counts
-  - `JobOperations.getTaskResult()` returns a `TaskCountsResult` object containing both task and slot counts
+  - `JobOperations.getTaskCountsResult()` returns a `TaskCountsResult` object containing both task and slot counts
+- Added property `requiredSlots` to `CloudTask`, allowing the user to specify how many slots on a node they should take up
+- **[Breaking]** Property `maxTasksPerNode` is replaced with `taskSlotsPerNode`, which allows nodes to consume a dynamic amount of slots for more fine-grained control over resource consumption
```

Comment:
Breaking changes are included in a separate section with `### Breaking changes` header. Also, please include which client API is impacted by this change.

Reply:
Fixed. Please verify. 😄